src/vivarium_gates_nutrition_optimization_child/data/utilities.py

```diff
@@ -156,6 +156,68 @@ def process_exposure(
     return data
 
 
+def process_gbd_2021_relative_risk(
+    data: pd.DataFrame,
+    key: str,
+    entity: Union[RiskFactor, AlternativeRiskFactor],
+    location: str,
+    gbd_round_id: int,
+    age_group_ids: List[int] = None,
+    whitelist_sids: bool = False,
+) -> pd.DataFrame:
+    # from vivarium_gbd_access.gbd.get_relative_risk
```

Comment:
It might be worth adding a comment about the differences between this and the standard rr processing function.